src/containers/Tablet/Tablet.tsx

```diff
@@ -74,18 +72,12 @@ export function Tablet() {
 
     const {id} = useParams<{id: string}>();
 
-    const [
-        {
-            nodeId: queryNodeId,
-            tenantName: queryDatabase,
-            type: queryTabletType,
-            clusterName: queryClusterName,
-        },
-    ] = useQueryParams(tabletQueryParams);
+    const [{tenantName: queryDatabase, type: queryTabletType, clusterName: queryClusterName}] =
```

Comment:
I've thought about this, but finally decided to leave tablet type in query string, cause as for me it doesn't hurt much: even if it will be incorrect, it will be replaced when tablet info is loaded. What do you think?

Reply:
I just don't like passing much in query strings, we already have very long links and they become even longer with every update. So for me, if there is no much difference, it's better not to pass

Reply:
Yeah, thats hurting truth! I agree with you! Lets get rid of query params if possible!